tools/bootstrap2pb.cc

```diff
@@ -0,0 +1,29 @@
+/**
+ * Utility to convert bootstrap from its YAML/JSON/proto representation to text
+ * proto.
+ *
+ * Usage:
+ *
+ * bootstrap2pb <input YAML/JSON/proto path> <output text proto path>
+ */
+#include <cstdlib>
+#include <fstream>
+
+#include "envoy/config/bootstrap/v2/bootstrap.pb.h"
+
+#include "common/common/assert.h"
+#include "common/protobuf/utility.h"
+
+// NOLINT(namespace-envoy)
+int main(int argc, char** argv) {
```

Comment:
We are going to end up needing to do basically the same thing for all of the xDS response fuzzing that we will also end up doing. Is it possible to make this program more general? Perhaps take a descriptor name on the CLI or something?

Reply:
Yeah, we should be able to do something like this using proto reflection, I'll look into it.

Reply:
FWIW, I think we'll catch the vast majority of bugs via the bootstrap fuzz. The ones we'll miss are the few places that the bootstrap isn't expressive enough, e.g. EDS only features.

Reply:
OK, it's not a big deal either way, just throwing it out there. Will approve and you can look into in follow up if you want.

Reply:
OK, will do followup PR.